Theme.ts

```diff
@@ -4,16 +4,19 @@ import type { DefaultTheme } from 'styled-components';
 export const AppTheme: DefaultTheme = {
   colours: {
     // Brand Colours
-    white: '#ffffff',
     cream: '#f9f2ec',
-    orange: '#ff7f30',
-    orange_dark: '#cc5e20',
-    blue: '#425cc7',
-    green: '#095157',
     purple: '#532b3a',
-    grey_dark: '#000000e6',
-    grey_light: '#3c3c3c',
-    black: '#00000033',
+    turquoise: '#57b6b2',
+    turquoise_dark: '#095157',
+    green: '#025157',
```

Comment:
I went with calling this teal so it matches their brand guide also purple --> plum

Reply:
It's quite dark for a teal, but that makes sense.